assignments/closure.js

```diff
@@ -1,11 +1,20 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
-
+function House() {
+  let occupants = 4;
+  this.howManyOccupants = function() {
+    console.log("There are " + occupants + " people living in this house.");
+  }
+}
+let myHouse = new House();
+//myHouse.howManyOccupants();
 // ==== Challenge 2: Create a counter function ====
-const counter = () => {
+const adder =(function () {
```

Comment:
Take a look at this code and compare to your solution, message me if you have any questions.
``` 
let adder = function() {
   let counter = 0;
   return function() {
      return counter += 1;
     };
 };

let newCounter = adder();

newCounter();//1
newCounter();//2
```